index.html

```diff
@@ -1876,6 +1876,76 @@ <h3>Examples</h3>
       </pre>
     </section>
   </section>
+  <section>
+    <h2>Camera Effect Status</h2>
```

Comment:
```suggestion
    <h2>Background Blur Effect Status</h2>
```

Reply:
The current proposal is scoped to blur, we can generalize later as additional effects are considered.